docs/manual/mkdocs.yml

```diff
@@ -250,6 +250,8 @@ nav:
         - o365group user add: 'cmd/graph/o365group/o365group-user-add.md'
         - o365group user list: 'cmd/graph/o365group/o365group-user-list.md'
         - o365group user remove: 'cmd/graph/o365group/o365group-user-remove.md'
+      - planner:
+        - task list: 'cmd/graph/planner/task-list.md'
```

Comment:
In the list of commands, we should use `planner task list`

Reply:
I fixed this, please check.